### ISSUE TITLE
ReferenceError: document is not defined

### ISSUE THREAD

**Issue by prplmark** [juliangarnier/anime#472]
3.0.0 no longer works in Next JS [v7.0.2].

`ReferenceError: document is not defined
    at Object.<anonymous> (/node_modules/animejs/lib/anime.js:919:1)`

Previously this was a non-issue, I'm left trying to figure out a way to dynamically import this without ssr. Looking forward to using these new features!

**Comment by prplmark** [comment]
I found a work-around that would work in React for now. I can require animejs after the component mounts:

```javascript
componentDidMount = () => {
     const anime = require("animejs/lib/anime.js");
}

**Comment by oklxq** [comment]
Same problem in vuepress  T_T!

**Comment by mattwagl** [comment]
Awesome! When I‘m back at my computer I‘ll send a PR to dev 😊. One last question: in order to test it from an application I had to create a build and commit it. Is this a problem? Should I create a clean branch without the build first? Thanks for the help.

**Comment by mattwagl** [comment]
I've removed the build and [prepared a PR](https://github.com/juliangarnier/anime/pull/481) with just the bugfix. Any chance this can make it into a patch release?

**Comment by juliangarnier** [comment]
Released in 3.0.1 !
